### PR TITLE
Changing CI explanation

### PR DESCRIPTION
**Pull request by ccamara**
### Summary

Readme states that CI is done using travis CI, and this is an outdated info. This minor, trivial, PR explains that we are using github actions instead.

### What should a reviewer concentrate their feedback on?

- [x] CI instructions are accurate and up-to-date.

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/r-devel/rdevguide/blob/main/README.md#contributors-) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/r-devel/rdevguide/blob/main/README.md#contributors-) in the README file:  ccamara
- [ ] Use the [How to acknowledge contributors](https://github.com/r-devel/rdevguide/blob/main/HOWTO-acknowledge-contributors.md) to add new contributor(s) using the [All Contributors](https://allcontributors.org/) Bot.

### CODE REVIEW

**Review by llrs — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ You can push directly to `main` for small fixes. Please use PRs to `main` for di
 
 ## Technical details
 
-Deployment is done via Travis CI:
+Deployment is done via GitHub Actions (as defined [here](https://github.com/r-devel/rdevguide/tree/main/.github/workflows)):
```
Comment (hturner) [inline README.md:33]:
```suggestion
Deployment is done via GitHub Actions, as specified in the [publish.yml](https://github.com/r-devel/rdevguide/blob/main/.github/workflows/publish.yml) workflow:
```

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ You can push directly to `main` for small fixes. Please use PRs to `main` for di
...
 
 - whenever there's a push to main, the book is built and its content is put in the `docs` directory.
```
Comment (hturner) [inline README.md:35]:
```suggestion
- whenever there's a push to main, the book is built and its content is put on the `gh-pages` branch, triggering the github-pages bot to deploy to <https://contributor.r-project.org/rdevguide/> .
```

**Review by hturner — CHANGES_REQUESTED** [review]
Thanks @ccamara, I corrected some of the technical details

**Comment by hturner** [comment]
@all-contributors please add @ccamara for documentation

**Review by hturner — APPROVED** [review]

**Pull request merged**